templates/emails/reviewers_group_invitation.rst

```diff
@@ -0,0 +1,18 @@
+Hello,
+
+We hope you are in good health and doing well.
+This email is in regards to PyCon India 2020 reviewer group. Pycon India is going virtual this year considering the current situations. We (conference organizers) take this as an opportunity to put more efforts in making this conference a great experience for attendees in other ways.
+
+We would like to put some more effort in bringing up some amazing talks to be presented this  Pycon 2020. We are creating a team of volunteers to review the talk proposals that should be presented in Pycon. Since this is a peer review process and driven by volunteers, we are looking for more volunteers to join us and help in making this a success. We believe you have great experience in Python and/or Python community and we would love if you can be a part of this group.
```

Comment:
```suggestion
Hence, we would like to put push towards bringing up some quality talks to be presented this  Pycon 2020. We are creating a team to review the talk proposals that should be presented in Pycon. Since this is a peer-review process and driven by volunteers, we are looking for folks like you to join us and help in making this a success. We believe your experience in Python and/or with Python community will be a core part of this team we intend to build going ahead. 
```

Make it personal to them for a better reach.

Reply:
```suggestion
Hence, we would like to push towards bringing in quality talks to be presented this PyCon 2020. We are creating a team to review the talk proposals that should be presented in PyCon. Since the talks are peer-reviewed and the process is driven by volunteers, we are looking for folks like you to join us and help in making this a success. We believe your experience with Python and the Python community will add vast value to the team we intend to build going ahead.
```